packages/cli/src/server/index.ts

```
@@ -110,9 +112,158 @@ export class AgentServer {
+        }
+      }
+
       // Initialize the database with await
-      await this.database.init();
-      logger.success('Database initialized successfully');
+      try {
+        await this.database.init();
+        logger.success('Database initialized successfully');
+      } catch (error) {
+        // Special handling for vector type error
+        if (error.message && error.message.includes('type "vector" does not exist')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)